0/rkm79791/p0.py

```diff
@@ -0,0 +1,96 @@
+#This is the first code I've ever written in python... maybe it works! 
+<<<<<<< HEAD
+#Ryan Maynard
+=======
+#By Ryan Maynard
+>>>>>>> eb020fabe0d3c916959d7f72da6bd4165d494b37
+import copy
+import numpy
+from masses import get_charge, get_mass
```

Comment:
masses.py is no in 0/rkm79791, so you'll need the following two lines:

import sys
sys.path.insert(0, '../../extra-files')

in order to be able to access the folder with masses.py

Reply:
Also keep in mind where the o-benzyne.xyz file is in relationship to p0.py